pkg/kubectl/cmd/get.go

```diff
@@ -355,6 +364,10 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			original = infos[ix].Object
 		}
 		if printer == nil || lastMapping == nil || mapping == nil || mapping.Resource != lastMapping.Resource {
+			if printer != nil {
+				w.Flush()
+				printer.FinishPrint(os.Stderr, lastMapping.Resource)
```

Comment:
It's for the case user requested to get several resource type at once. So after one resource type list is ended we should check whether there's something to add in the end (look at "kubectl get pods,namespaces" in output examples above in PR description).